src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -101,29 +102,32 @@ public void testVCFGZIndex() {
     }
 
     @Test
-    public void testVCFGZIndex_inferredName_onlyCreate() {
+    public void testVCFGZIndex_tabix() {
         final File ORIG_FILE = new File(getToolTestDataDir(), "test_variants_for_index.vcf.blockgz.gz"); //made by bgzip
+        final File outName = BaseTest.createTempFile("test_variants_for_index.blockgz.gz.", TabixUtils.STANDARD_INDEX_EXTENSION);
+
         final String[] args = {
-                "--feature_file", ORIG_FILE.getAbsolutePath(),
+                "--feature_file" ,  ORIG_FILE.getAbsolutePath(),
+                "-O" ,  outName.getAbsolutePath()
         };
         final Object res = this.runCommandLine(args);
-        final File tabixIndex = new File(ORIG_FILE.getAbsolutePath() + IndexFeatureFile.TABIX_INDEX_EXTENSION);
-        Assert.assertEquals(res, tabixIndex.getAbsolutePath());
-        tabixIndex.deleteOnExit();
+        Assert.assertEquals(res, outName.getAbsolutePath());
 
-        Assert.assertTrue(tabixIndex.exists(), tabixIndex + " does not exists");
+        final Index index = IndexFactory.loadIndex(res.toString());
+        Assert.assertTrue(index instanceof TabixIndex);
 
-        //NOTE: We can create this index but we can't read it (see  testVCFGZIndex_inferredName)
+        Assert.assertEquals(index.getSequenceNames(), Arrays.asList("1", "2", "3", "4"));
+        checkIndex(index, Arrays.asList("1", "2", "3", "4"));
     }
 
-    @Test(enabled = false) //TODO enable when this is fixed: https://github.com/broadinstitute/gatk/issues/1187
+    @Test
     public void testVCFGZIndex_inferredName() throws IOException {
         final File ORIG_FILE = new File(getToolTestDataDir(), "test_variants_for_index.vcf.blockgz.gz"); //made by bgzip
         final String[] args = {
                 "--feature_file" ,  ORIG_FILE.getAbsolutePath(),
         };
         final Object res = this.runCommandLine(args);
-        final File tabixIndex = new File(ORIG_FILE.getAbsolutePath() + IndexFeatureFile.TABIX_INDEX_EXTENSION);
+        final File tabixIndex = new File(ORIG_FILE.getAbsolutePath() + TabixUtils.STANDARD_INDEX_EXTENSION);
         Assert.assertEquals(res, tabixIndex.getAbsolutePath());
         tabixIndex.deleteOnExit();
 
```

Comment:
Delete the `//BUG` comments from the lines below, and add a call to `checkIndex()` like in `testVCFGZIndex_tabix()`